routes/v1.js

```diff
@@ -1040,18 +1063,19 @@ api.declare({
   // Construct status object
   var status = task.status();
 
-  // Publish message about last run, if it was canceled
+  // If the last run was canceled, resolve dependencies and publish message
   if (run.state === 'exception' && run.reasonResolved === 'canceled') {
```

Comment:
`canceled` is a good reason to think about enum, imho. [I always want to spell it with two l's](http://grammarist.com/spelling/cancel/).